src/lazy/binary/raw/v1_1/reader.rs

```diff
@@ -300,4 +300,59 @@ mod tests {
 
         Ok(())
     }
+
+    macro_rules! assert_closely_eq {
+        ($x:expr, $y:expr, $d:expr) => {
+            let (a, b) = ($x, $y);
+            if ((a - b).abs() >= $d) {
+                panic!(
+                    "float not close enough to expected value: left: {}, right: {}",
+                    a, b
+                );
+            }
+        };
+    }
```

Comment:
For serialization/deserialization, I don't think we should be considering any epsilon value. A round trip of a float value should result in a value that is equivalent to the original value, and (for non-NaN values) I that means we should expect actual equality of the actual and expected values.

Reply:
+1. If roundtripping from `f64 -> f32 -> f64` results in a different value, then `f32` isn't a valid encoding of that value. You can use `IonEq::ion_eq` to compare floats with the expected `NaN` handling.

Reply:
Ah yea, thank you for calling this out. I had originally intended to round-trip "friendly" float literals and use this so I wasn't using the true float value for comparison. I didn't go that route, and forgot to remove this.